deploy/kubernetes/examples/demo-snapshotclass-using-secret-from-configuration.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: demo-snapshotclass
+driver: block.csi.ibm.com
+deletionPolicy: Delete
+parameters:
+  # pool, SpaceEfficiency and snapshot_name_prefix can be provided in by_system_id parameter for each system or by parameter name as default.
```

Comment:
`pool, SpaceEfficiency and snapshot_name_prefix` could we just refer to them as something like "non-secret parameters"?

Reply:
I prefer having the explicit way since we have more non-secret parameters to which the comment isn't related. @ArbelNathan WDYT?

Reply:
@rivkap what do you think?

Reply:
@zingero which non-secret parameters are not relevant to the comment?

Reply:
@ArbelNathan @oriyarde 
It depends on what the other parameters are that are not relevant, as you asked above.
In any case - 
1. If we keep this note we should add a comma
      # pool, SpaceEfficiency**,** and snapshot_name_prefix

2. We can also write non-secret parameters in the example file itself and always add a note / text in the documentation itself with more specific information (ie. the list of what parameters can be provided)

Reply:
@oriyarde your suggestion doesn't refer to not relevant parameters, but all non-secret parameters.

Reply:
@zingero what is the difference? why can't we refer to all possible `by_system_id` parameters as "non-secret parameters"?

Reply:
@oriyarde because parameters like `csi.storage.k8s.io/snapshotter-secret-name` and `csi.storage.k8s.io/snapshotter-secret-namespace` are secret-related, but the comment doesn't refer to these parameters.

Reply:
then how about "non-secret-related parameters"?

Reply:
Sounds great

Reply:
done

Reply:
why without hyphen? "non" as a separate word is [obsolete](https://en.wiktionary.org/wiki/non#English) and [used only in phrases from Latin](https://en.wiktionary.org/wiki/non-#Usage_notes)

Reply:
@zingero is the "related" word really necessary?

Reply:
suggestion:
`can be provided in by_system_id parameter for each system or by parameter name as default.` ->

1. `can be specified in by_system_id per system, or outside by_system_id as cross-system default.`
2. `can be specified in by_system_id per system, to override cross-system parameters.`
3. `can be specified in by_system_id per system, to override parameters outside by_system_id.``

Reply:
I just realized that fstype is not secret related, but is also a sidecar parameter :/

Reply:
then I suggest: `non-csi.storage.k8s.io parameters`

Reply:
no, it's not necessary

Reply:
I agree with the new name

Reply:
so my current full suggestion:
`non-csi.storage.k8s.io parameters may be specified in by_system_id per system, or outside by_system_id as cross-system default.`

Reply:
👍🏻

Reply:
ok, done

Reply:
perhaps the `or` should be `and/or` - I hope it clarifies which one overrides the other.
@rivkap what do you think?

Reply:
@oriyarde 
I suggest:
`non-csi.storage.k8s.io parameters may be specified in by_system_id per system and/or outside by_system_id as the cross-system default. `

Reply:
thanks @rivkap

Reply:
done